apollo-ios/Sources/Apollo/MultipartResponseDeferParser.swift

```diff
@@ -59,7 +59,7 @@ struct MultipartResponseDeferParser: MultipartResponseSpecificationParser {
     for dataLine in chunk.components(separatedBy: Self.dataLineSeparator.description) {
       switch DataLine(dataLine.trimmingCharacters(in: .newlines)) {
       case let .contentHeader(type):
-        guard type == "application/json" else {
+        guard type == "application/graphql-response+json" || type == "application/json" else {
```

Comment:
We should create a helper method for this. We're doing string comparisons against strings that are really constants. And we're doing the same comparison in two places. This is prone to future bugs.

 We should store the constants as `StaticString`s and have a helper method to check the content header.

Reply:
I'll put this PR back into draft and make that change when I get back.